src/index.ts

```diff
@@ -7,7 +7,7 @@ import {NightwatchInitiator} from '@nightwatch/setup-tools';
 import {NIGHTWATCH_TITLE, AVAILABLE_CONFIG_FLAGS} from './constants';
 import Logger from './logger';
 import minimist from 'minimist';
-import suggestSimilarOption from './utils/suggestSimilar';
```

Comment:
Please remove `./utils/suggestSimilar`. Also, what's the motive behind this change?
Also, why not use [didyoumean](https://www.npmjs.com/package/didyoumean) or [didyoumean2](https://www.npmjs.com/package/didyoumean2)?

Reply:
@vaibhavsingh97 thank you for your review, it was suggested that `didyoumean` library be used in place of the package `suggestSimilarOption` for suggesting Nightwatch flags when a wrong flags is been detected and then terminate the Nightwatch process. I will refactor the changes now with `didyoumean` and then remove the `suggestSimilarOption`.

Reply:
@vaibhavsingh97 i trust you are doing very well today, i wanted to mentioned that i have updated this PR with your suggestions above and now the PR has also passed the required check but could you please help look at this when you have some moment.